app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
```

Comment:
- [ ] Please let's match the styling of the wireframe for the login page. Your login page is aligned to the left instead of being centered. The wireframe design is shown below 👇 
### Wireframe
![image](https://user-images.githubusercontent.com/46839548/186241253-f9c58a25-af4a-4b15-a273-60cdeba9b83f.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/186241473-e302e518-0adb-4ddd-85c8-a0e5a8ef1933.png)

- [ ] Kindly ensure to do the same for your `sign_up` page